pkgs/tools/misc/copier/default.nix

```diff
@@ -0,0 +1,61 @@
+{ lib, gnused, poetry, git, python3, fetchFromGitHub }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "copier";
+  version = "unstable-2021-11-10";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "copier-org";
+    repo = "copier";
+    rev = "9f05fdf77d725933be3f779f1c3215467851f92a";
+    sha256 = "sha256-uUM9Zzh2uOEqcwZxubjj4PawHmqZhQDFe7aVQKe6lM8=";
+  };
+
+  postPatch = ''
+    ${gnused}/bin/sed -e '/packaging/d' -i pyproject.toml
+  '';
+
+  nativeBuildInputs = [
+    poetry
```

Comment:
Upstream seems to use `poetry-core` as [build backend](https://github.com/copier-org/copier/blob/master/pyproject.toml#L120).